exasol_transformers_extension/resources/templates/model_downloader_udf.jinja.sql

```diff
@@ -0,0 +1,9 @@
+CREATE OR REPLACE {{ language_alias }} SCALAR SCRIPT "TE_MODEL_DOWNLOADER_UDF"(
+    model_name VARCHAR(2000000),
+    path_in_the_bucket VARCHAR(2000000),
```

Comment:
I wouldn't specify the path in bucket via an extra parameter, I would push this all into the bfs_con, because I think, we can assume the following scenario, an admin creates the bfs connection object and defines where the cache should be and the user get access granted for this connection.